README.md

```diff
@@ -1 +1,29 @@
 # sync-templates
+
+[![Continuous Integration](https://github.com/fiberplane/sync-templates/actions/workflows/ci.yml/badge.svg)](https://github.com/fiberplane/sync-templates/actions/workflows/ci.yml)
+
+The `fiberplane/sync-templates` action is a utility that syncs your Fiberplane Templates from a designated directory in your repository with your Fiberplane Workspace.
+
+## Usage
+This action can be run on `ubuntu-latest` and `macos-latest` GitHub Actions runners.
+
+A minimum working example of the action:
+```yaml
+steps:
+- uses: fiberplane/sync-templates@v1
+  with:
+    api-token: ${{ secrets.FP_TOKEN }} # it is best practice to keep your secrets in GitHub Secrets
+    workspace-id: ${{ secrets.FP_WORKSPACE_ID }}
```

Comment:
not necessarily but I would still suggest keep it somewhat concealed maybe?

Reply:
I think having it as a "normal" variable is better, since it might help with debugging. But this way the example is a bit simpler....

Reply:
yea i'd also agree that a normal variable would be better. it would also be consistent with our other action then: https://github.com/marketplace/actions/publish-event

Reply:
ok! Will change it to that